savant/deepstream/element_factory.py

```diff
@@ -0,0 +1,59 @@
+"""DeepStream pipeline elements factory."""
+from dataclasses import replace
+import pyds
+from savant.config.schema import PipelineElement, PyFuncElement
+from savant.gstreamer import Gst  # noqa:F401
+from savant.gstreamer.element_factory import GstElementFactory
+from savant.utils.platform import is_aarch64
+
+
+class NvDsElementFactory(GstElementFactory):
+    """Creates pipeline elements."""
+
+    def create(self, element: PipelineElement) -> Gst.Element:
+        """Creates specified element.
+
+        :param element: Pipeline element to create.
+        :raises CreateElementException: Unknown element.
+        :return: Gst.Element.
+        """
+        # if isinstance(element, PyFuncElement):
+        #     return self.create_pyfunc_bin(element)
```

Comment:
Please add a comment why we need `create_pyfunc_bin` and why we cannot use it now.